src/Raven.Server/Documents/CountersStorage.cs

```diff
@@ -495,59 +831,29 @@ public IEnumerable<(string ChangeVector, long Value)> GetCounterValues(Documents
         {
```

Comment:
We can probably abandon the IEnumerable return type for this function and for GetCountersForDocument